code/modules/cargo/packs.dm

```diff
@@ -256,12 +254,26 @@
 /datum/supply_pack/emergency/spacesuit
 	name = "Space Suit Crate"
 	desc = "Contains one aging suit from Space-Goodwill and a jetpack. Requires EVA access to open."
-	cost = 2500
-	access = ACCESS_EVA
+	cost = 900
+	contains = list(/obj/item/clothing/suit/space,
+					/obj/item/clothing/head/helmet/space,
+					/obj/item/clothing/mask/breath)
+	crate_name = "space suit crate"
+	crate_type = /obj/structure/closet/crate/secure
+
+/datum/supply_pack/emergency/spacesuit/bulk
+	name = "Space Suit Crate"
+	desc = "Contains three aging suits from Space-Goodwill and a jetpack. Requires EVA access to open."
+	cost = 1900
```

Comment:
That's a pretty big discount considering 1 suit is 900

Reply:
1 suit is 500, considering the value of the manifest its not actually a discount at all.

Well, the discount is in the time you save by not having to move 3 crates and approve 3 manifests.